pkg/cmd/install/addons/scenario/addons/appmgr/clusterrole.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: multicloud-operators
+  name: multicluster-operators
```

Comment:
change this to a more specific name. I think it is only used for subscription

Reply:
You are right. Done as suggested, changed all `multicluster-operators` references to `multicluster-operators-subscription`. Thanks.